### PR TITLE
Adds transformation rules

### PR DESCRIPTION
**Pull request by mrdunski**

### CODE REVIEW

**Review by techt3 — CHANGES_REQUESTED** [review]
cant run unit tests:
```
# github.com/prometheus/snmp_exporter/collector
/home/wilk/src/snmp_exporter/collector/gosnmp_logger_wrapper.go:18: missing ... in args forwarded to printf-like function
FAIL	github.com/prometheus/snmp_exporter/collector [build failed]
```

File: config/config.go [hunk 1]
```diff
@@ -71,10 +72,11 @@ type WalkParams struct {
...
 type Module struct {
 	// A list of OIDs.
-	Walk       []string   `yaml:"walk,omitempty"`
-	Get        []string   `yaml:"get,omitempty"`
-	Metrics    []*Metric  `yaml:"metrics"`
-	WalkParams WalkParams `yaml:",inline"`
+	Walk       []string        `yaml:"walk,omitempty"`
+	Get        []string        `yaml:"get,omitempty"`
+	Metrics    []*Metric       `yaml:"metrics"`
+	WalkParams WalkParams      `yaml:",inline"`
```
Comment (techt3) [inline config/config.go:78]: I'm not sure about this leading ",", that's not a mistake?

**Review by techt3 — APPROVED** [review]
I suggest adding a GitHub action that will run all unit tests.

File: config/config.go (hunk 1 above)
Comment (mrdunski) [reply config/config.go:78]: I didn't really change that but I don't think it is. Without it the annotation would mean the name instead the option.

**Review by antonipbednarskittpsc — APPROVED** [review]

**Review by robsztal — APPROVED** [review]

**Pull request merged**